### PR TITLE
Include <QtGui/qtgui-config.h> to fix flatpak builds

### PR DESCRIPTION
**Pull request by daschuer**
This aims to fix https://github.com/mixxxdj/mixxx/issues/12802

### CODE REVIEW

**Comment by daschuer** [comment]
@djantti can you confirm this fixes the build error for you? Than let's merge this small regression fix to 2.4.

**Comment by daschuer** [comment]
@Be-ing has confirmed this fixes the flatpack buil error.  
This low risk and can be merged to the 2.4.0 release candidate.

File: src/waveform/renderers/glwaveformrenderer.h [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <QtGui/qtguiglobal.h> // for QT_NO_OPENGL and QT_OPENGL_ES_2
```
Comment (Be-ing) [inline src/waveform/renderers/glwaveformrenderer.h:3]:
```suggestion
// This is where QT_OPENGL_ES_2 may be defined.
#include <QtGui/qtgui-config.h>
```

This more directly gets at what is needed here. I have confirmed it still builds an aarch64 Flatpak.

**Comment by daschuer** [comment]
Done

**Review by Be-ing — APPROVED** [review]

**Pull request merged**